apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SqlResultStream.RetryStateTests.cs

```diff
@@ -153,7 +165,7 @@ public async Task RecordErrorAndWait_RetryInfo()
             var exception1 = new RpcException(new Status(StatusCode.Unavailable, "Bang"), trailers);
             var exception2 = new RpcException(new Status(StatusCode.Unavailable, "Bang"));
 
-            RetryState state = new RetryState(mock.Object, settings, RetrySettings.NoJitter, maxConsecutiveErrors: 5);
+            RetryState state = new RetryState(mock.Object, s_retrySettings, maxConsecutiveErrors: 5);
```

Comment:
Just flagging that the old `settings` used here are not the same as `s_retrySettings`, which might not be a problem anyways.

Reply:
In what way? Both are initially 1s, max 5s, multiplier 2.0, no jitter.

Reply:
I think s_retrySettings are max 15s.